app/Utils.php

```diff
@@ -128,4 +128,18 @@ public static function saveToFile(string $pathDirectory, $content, string $comme
 		}
 		return (bool) $value;
 	}
+
+	/**
+	 * Add column.
+	 *
+	 * @param string                      $tableName
+	 * @param string                      $columnName
+	 * @param \yii\db\ColumnSchemaBuilder $columnSchema
+	 *
+	 * @return bool
+	 */
+	public static function addColumn(string $tableName, string $columnName, \yii\db\ColumnSchemaBuilder $columnSchema)
```

Comment:
this function already exists 
https://github.com/YetiForceCompany/YetiForceCRM/blob/5daa4864226079d9908e9cb724971ec32728f0d3/app/Db.php#L228-L234